### PR TITLE
Added alternative unit minutes/km to speed in addition to km/h #130

### PR DESCRIPTION
**Pull request by abarber7**

### CODE REVIEW

File: src/test/java/app/gpx_animator/KmHrMinPerKmTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                return String.format("%d km/h | " + (int) mins + ":00 minutes/km", speed); //NON-NLS
+            } else if (secs >= tenS) {
+                return String.format("%d km/h | " + (int) mins + ":" + (int) secs + " minutes/km", speed); //NON-NLS
+            } else {
+                return String.format("%d km/h | " + (int) mins + ":0" + (int) secs + " minutes/km", speed); //NON-NLS
+            }
+        } else {
+            return String.format("%d km/h | 0:00 minutes/km", speed); //NON-NLS
+        }
+    }
```
Comment (McPringle) [inline src/test/java/app/gpx_animator/KmHrMinPerKmTest.java:26]: This method looks identical to the same method in the ´Renderer´ class. In this case you test the method in the test class, not the method in the ´Renderer´ class.

**Comment by McPringle** [comment]
Thank you very much for your PR, @abarber7. I would like to see an option in the GUI and on the command line to define the unit. There could be more than these two, like "mph" or "nm" (nautical miles). I'll merge your PR, do some refactorings and add the UI stuff.

**Comment by abarber7** [comment]
Hello,
I have Added "mph" and a "nm" (nautical miles) option.

Added alternative unit minutes/km to speed in addition to km/hr #130
Added alternative unit minutes/mile to speed in addition to mile/hr #130
Added alternative unit minutes/nm to speed in addition to nm/hr #130

**Comment by McPringle** [comment]
Thank you very much for your work!

**Comment by McPringle** [comment]
The ´Renderer´ class is very huge. So I decided to extract the speed calculation to a separate ´SpeedUtil´ class. With time, I hope that the ´Renderer´class will become clearer. Second, I created a ´SpeedUnit´ enum to collect all the speed units. That makes it easier to add a new unit (just one place to add it).

Again, thank you very much for your contribution. :-)

**Pull request merged**